roles/docker/tasks/package_docker.yml

```diff
@@ -48,7 +48,9 @@
     template:
       dest: "{{ docker_systemd_dir }}/custom.conf"
       src: custom.conf.j2
-  when: not os_firewall_use_firewalld | default(False) | bool
+    when: not (os_firewall_use_firewalld | default(False)) | bool
```

Comment:
I think this when condition was meant to be at the block level. If we're demoting it to just this task then lets get rid of the block otherwise lets move it up to the block definition.

Reply:
Good catch.